src/main/java/io/split/android/client/track/TrackStorageManager.java

```diff
@@ -57,12 +71,21 @@ public void close(){
         saveToDisk();
     }
 
-    private void loadEventsFromDisk(){
+    private void loadEventsFromDisk() {
+        if(mFileStorageManager.exists(CHUNK_HEADERS_FILE_NAME)) {
+            loadEventsFromMultipleFiles();
+        } else {
+            loadEventsFromOnFile();
+        }
+    }
 
+    private void loadEventsFromOnFile() {
+        // Legacy file
         try {
-            String storedTracks = mFileStorageManager.read(EVENTS_FILE_NAME);
-            if(Strings.isNullOrEmpty(storedTracks)) {
+            String storedTracks = mFileStorageManager.read(LEGACY_EVENTS_FILE_NAME);
+            if (Strings.isNullOrEmpty(storedTracks)) {
```

Comment:
minor: I noticed spaces after the `if` statement and other places without. For example this line 86 vs line 75 of this file in this PR. It would be nice to have a common criteria.

Reply:
Yup, fixed. It would be good idea to add a linter to avoid this kind of issues.